docs/ReleaseNotes.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+## v0.9 
+Excited to announce the release of admiral version `v0.9` with lots of cool functionality added and it is ready for production usage. Following are the highlights of this release:
+
+`We would like to thank all the contributors and everyone who played a role in testing the aplha and beta releases of admiral.`
+
+### [Global Traffic Policies](./Architecture.md#Global Traffic Policy)
+Global traffic policies allow defining custom traffic routing behaviour for a admiral generated CNAME, for example routing all traffic for a service to uswest region. This feature relies on Istio's [locality load balancing](https://istio.io/docs/ops/configuration/traffic-management/locality-load-balancing/)
```

Comment:
... for admiral generated CNAMEs ...

Is there an example that doesn't pin us to AWS?

Reply:
I have updated the text to generic and say region and AZ instead of uswest-2. No we do not have an example that is not aws, we can put some effort to do i) non aws and ii) hybrid cloud multi-cluster examples.